scan-command/src/main/java/io/ballerina/scan/internal/StaticCodeAnalyzer.java

```diff
@@ -72,6 +80,70 @@ public void visit(CheckExpressionNode checkExpressionNode) {
         }
     }
 
+    public void visit(BinaryExpressionNode binaryExpressionNode) {
+        if (binaryExpressionNode.operator().kind().equals(ELLIPSIS_TOKEN)
```

Comment:
Can extract `binaryExpressionNode.operator()` and then the kind to variables to avoid repeated method calls.

Please check and refactor the code across the PRs. Let's make sure we're following best practices.